src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala

```diff
@@ -242,7 +291,12 @@ object JsonSchemaGenerator {
      * @return JsonSchemaType with recognized properties
      */
     def enrichString(value: String) = {
-      guessFormat(value, formatSuggestions) match {
```

Comment:
@cevaris ok, we'll look if it works for us. Thank you again. Any other contributions would be highly appreciated.